src/__tests__/VoucherRedemption.tsx

```diff
@@ -0,0 +1,263 @@
+    owner_image_url: 'assets/shunfa-bakery-logo.png',
+    created_at: '2020-04-15T07:12:08.556Z',
+    updated_at: '2020-04-25T20:06:37.041Z',
+    target_amount: 1000000,
+    summary: null,
+    hero_image_url: 'assets/shunfa-bakery-hero.png',
+    progress_bar_color: null,
+    business_type: 'Family-owned',
+    num_employees: 3,
+    founded_year: 2015,
+    website_url: null,
+    menu_url: null,
+    square_location_id: '3D0QAW4CSZJWZ',
+    locations: [
+      {
+        id: 1,
+        address1: '6221 Fort Hamilton Pkwy',
+        address2: null,
+        city: 'Brooklyn',
+        state: 'NY',
+        zip_code: '11219',
+        seller_id: 35,
+        created_at: '2020-04-15T07:14:16.501Z',
+        updated_at: '2020-04-15T07:14:16.501Z',
+        phone_number: '(718) 833-8884',
+      },
+    ],
+    amount_raised: 93500,
+  };
+
+  const voucherResponse: any = {
+    id: 4,
+    seller_id: 'shunfa-bakery',
+    item_type: null,
+    created_at: '2020-07-19T23:15:00.544Z',
+    updated_at: '2020-07-19T23:15:00.544Z',
+    payment_intent_id: 4,
+    refunded: false,
+    purchaser_id: 1,
+    gift_card_detail: {
+      id: 1,
+      gift_card_id: '123asdf',
+      receipt_id: null,
+      expiration: null,
+      created_at: '2020-07-19T23:15:00.582Z',
+      updated_at: '2020-07-19T23:15:00.582Z',
+      item_id: 4,
+      seller_gift_card_id: '123asdf',
+      recipient_id: 1,
+      single_use: false,
+      amount: 2500,
+    },
+  };
+
+  const voucherUpdateResponse: any = {
+    id: 4,
+    seller_id: 'shunfa-bakery',
+    item_type: null,
+    created_at: '2020-07-19T23:15:00.544Z',
+    updated_at: '2020-07-19T23:15:00.544Z',
+    payment_intent_id: 4,
+    refunded: false,
+    purchaser_id: 1,
+    gift_card_detail: {
+      id: 1,
+      gift_card_id:
+        'rphtgeghnxtenpficladyeaklccdkqmxldcshuodqpvaujlshfoxizldomrhpbje',
+      receipt_id: null,
+      expiration: null,
+      created_at: '2020-07-19T23:15:00.582Z',
+      updated_at: '2020-07-19T23:15:00.582Z',
+      item_id: 4,
+      seller_gift_card_id:
+        'crlmuwcorvfwrhnzdfkdjkjhwwitqqwnhhelyahrctehykfustgvrbalipvwiaus',
+      recipient_id: 1,
+      single_use: false,
+      amount: 300,
+    },
+  };
+
+  return {
+    ...jest.requireActual('../utilities/api/interactionManager'),
+    getSeller: jest.fn(() => ({
+      data: shunfaBakeryResponse,
+    })),
+    getVoucher: jest.fn(() => ({
+      data: voucherResponse,
+    })),
+    updateVoucher: jest.fn(() => ({
+      data: voucherUpdateResponse,
+    })),
+  };
+});
+
+beforeAll(() => {
+  axios.defaults.adapter = require('axios/lib/adapters/http');
+});
+
+afterEach(() => cleanup());
+
+const renderVoucher = (providerValues) => (
+  <VoucherProvider>
+    <VoucherPage />
+  </VoucherProvider>
+);
+
+describe('Voucher Page', () => {
+  let providerValues = {};
+  let VoucherProps = {};
+  let VoucherDispatcher = jest.fn();
+  beforeEach(() => {
+    VoucherProps = VoucherTypes.defaultState;
```

Comment:
Is it normal for these to be capitalized?

Reply:
I did it to match the type of capitalizing i did with the the react hook, i dont know the exact standard on naming for those.